src/test/java/Task30/LoginTest.java

```diff
@@ -1,61 +1,33 @@
 package Task30;
 
-import io.github.bonigarcia.wdm.WebDriverManager;
-import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeEach;
+import page.LoginPage;
+import page.MailPage;
 import org.junit.jupiter.api.Test;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
-
-import java.util.concurrent.TimeUnit;
-import static Task30.Locators.*;
-
-public class LoginTest {
 
+public class LoginTest extends BaseTest{
+    private WebDriver driver;
     public static final String USERNAME = "sea123alena";
     public static final String PASSWORD = "Bazarevsky2";
-    private WebDriver driver;
 
-    @BeforeEach
-    void setUp() {
-        WebDriverManager.chromedriver().setup();
-        driver = new ChromeDriver();
-        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+    @Test
+    public void loginToYandexMailTest(){
 
-        driver.manage().window().maximize();
-        driver.get(SITE_NAME);
+        LoginPage loginPage = new LoginPage();
+        loginPage.loginToYandex(USERNAME, PASSWORD);
+        Assertions.assertTrue(loginPage.isInboxDisplayed());
     }
-
-
     @Test
-    public void loginTest() throws InterruptedException {
+    public void logoutFromMailTest(){
 
-        WebElement login = driver.findElement(LOGIN_BUTTON);
-        login.click();
-
-        WebElement userName = driver.findElement(USERNAME_FIELD);
-        userName.sendKeys(USERNAME);
-
-        driver.findElement(SUBMIT_BUTTON).click();
-        Thread.sleep(5000 );//implicity waiting
-
-        WebElement password = driver.findElement(PASSWORD_FIELD);
-        password.sendKeys(PASSWORD);
-
-        driver.findElement(SUBMIT_BUTTON).click();
-
-        new WebDriverWait(driver, 3).until(ExpectedConditions.visibilityOfElementLocated(USER_NAME));
-
-        Assertions.assertTrue(driver.findElement(INBOX).isDisplayed());
-    }
+        LoginPage loginPage = new LoginPage();
+        loginPage.loginToYandex(USERNAME, PASSWORD);
 
-    @AfterEach
-    public void cleanUp() {
-        driver.quit();
+        Assertions.assertTrue(loginPage.isInboxDisplayed());
+        MailPage logoutPage = new MailPage();
+        logoutPage.clickLogout();
+        Assertions.assertTrue(logoutPage.isLoginButtonDisplayed());
```

Comment:
your tests may look like:
```
public class LoginTest extends BaseTest {

    public static final String USERNAME = "sea123alena";
    public static final String PASSWORD = "Bazarevsky2";

    private LoginPage loginPage;
    private MailPage mailPage;

    @BeforeEach
    public void login() {
        loginPage = new LoginPage();
    }

    @Test
    public void loginTest() {
        mailPage = loginPage.loginToYandex(USERNAME, PASSWORD);
        Assertions.assertTrue(loginPage.isInboxDisplayed());
    }

    @Test
    public void logoutFromMailTest() {
        mailPage = loginPage.loginToYandex(USERNAME, PASSWORD);
        loginPage = mailPage.clickLogout();
        Assertions.assertTrue(loginPage.isLoginButtonDisplayed());
    }
}
```

and even `mailPage = loginPage.loginToYandex(USERNAME, PASSWORD);` you may move into @BeforeEach